tensorpack/dataflow/imgaug/geometry.py

```diff
@@ -43,17 +72,21 @@ class RotationAndCropValid(ImageAugmentor):
         Note that this will produce images of different shapes.
     """
 
-    def __init__(self, max_deg, interp=cv2.INTER_LINEAR):
+    def __init__(self, max_deg, interp=cv2.INTER_LINEAR, step_deg=0):
```

Comment:
No need to add this feature again. It will be equivalent to Rotation (because there is no need to crop when rotating 90 degrees).